tests/test_openNeuroClient.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import os
+import time
+from rtCommon.structDict import StructDict
+from rtCommon.clientInterface import ClientInterface
+from projects.openNeuroClient import openNeuroClient
+from tests.backgroundTestServers import BackgroundTestServers
+from tests.common import testPath, rtCloudPath
+
+tmpDir = os.path.join(testPath, 'tmp/')
+openNeuroClientProjectPath = os.path.join(rtCloudPath, 'projects/openNeuroClient')
+
+allowedDirs =  ['/tmp', openNeuroClientProjectPath]
```

Comment:
As with earlier comments, for this usage and the one a few lines above, I think `tempfile` for a temp directory would be better